app/preferences.go

```diff
@@ -64,19 +64,22 @@ func (p *preferences) load() {
 	}
 }
 
-func (p *preferences) loadFromFile(path string) error {
+func (p *preferences) loadFromFile(path string) (err error) {
```

Comment:
Naming the return in this case is slightly confusing as it is shadowed completely by the next line

Reply:
It doesn't work if you remove the named return